src/user/signals.py

```diff
@@ -150,45 +150,6 @@ def create_action(sender, instance, created, **kwargs):
                     )
             user = instance.created_by
 
-        """
-        If we're creating an action for the first time,
-        check if we've been referred
-        """
-        referral_content_types = [
```

Comment:
This is old referral program logic where we would pay when invited user created their first contribution